### ISSUE TITLE
Issue with advanced and Basic Boilerplate

### ISSUE THREAD

**Issue by gcid12** [graphql-boilerplates/react-fullstack-graphql#329]
This is very similar to #319 
When bootstrapping the advanced boilerplate with graphql create myapp -b react-fullstack-advanced and following the instructions to start the app:
``` 
cd myapp/server
yarn start 
```
it shows:

``` 
Error: Cannot find module 'graphql-yoga'
```

This also happens with the basic boilerplate

**Comment by yakotika** [comment]
I have the same problem

**Comment by AesSedai** [comment]
Same issue here.

```
$ yarn start
yarn run v1.6.0
warning package.json: No license field
$ nodemon -e js,graphql -x node -r dotenv/config src/index.js
[nodemon] 1.17.3
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node -r dotenv/config src/index.js`
module.js:549
    throw err;
    ^

Error: Cannot find module 'graphql-yoga'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/zrlovin/Development/et2/server/src/index.js:1:89)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by MinJieLiu** [comment]
I have the same problem too.

**Comment by marktani** [comment]
This problem was [fixed in graphql-yoga@1.13.1](https://github.com/graphcool/graphql-yoga/pull/299), can you upgrade your `graphql-yoga` version and try again?

I'll close this issue, as all boilerplates have been upgraded to `1.13.1` 🙂
